doc/tutorials/quickstart_in_rviz/CMakeLists.txt

```diff
@@ -1 +1,24 @@
 install(DIRECTORY launch DESTINATION share/${PROJECT_NAME})
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  find_package(ament_cmake_gtest REQUIRED)
+  find_package(ros_testing REQUIRED)
+  find_package(Boost REQUIRED COMPONENTS filesystem)
+
+  # These don't pass yet, disable them for now
+  set(ament_cmake_copyright_FOUND TRUE)
```

Comment:
Why doesn't the copyright test pass? Shouldn't that be easy to fix? Add a license file?

Reply:
I just copy/pasted this from somewhere else, but actually all of these tests pass. :+1: Removed it.